mlrun/api/crud/client_spec.py

```diff
@@ -26,6 +26,14 @@ def get_client_spec(self):
             dask_kfp_image=config.dask_kfp_image,
             api_url=config.httpdb.api_url,
             nuclio_version=self._resolve_nuclio_version(),
+            # These don't have a default value, but we don't send them if they are not set
+            # because they will be irrelevant
```

Comment:
```suggestion
            # These don't have a default value, but we don't send them if they are not set to allow the client to know when to use server value and when to use client value (server only if set)
            # Since their default value is empty and not set is also empty we can use the same _get_config_value_if_not_default
```